govc/cli/command.go

```diff
@@ -65,14 +65,14 @@ It is licensed under the Apache License, Version 2.0
 
 The available commands are listed below. A detailed description of each
 command can be displayed with "govc <COMMAND> -h". The description of all
-commands can be also found at https://via.vmw.com/GJ98hk .
+commands can be also found at https://github.com/vmware/govmomi/blob/main/govc/USAGE.md.
 
 Examples:
   show usage of a command:       govc <COMMAND> -h
   show toplevel structure:       govc ls
   show datacenter summary:       govc datacenter.info
   show all VMs:                  govc find -type m
-  upload a ISO file:             govc datastore.upload -ds datastore1 ./config.iso vm-name/config.iso
+  upload a ISO file:              govc datastore.upload -ds datastore1 ./config.iso vm-name/config.iso
```

Comment:
Thanks @tenthirtyam , can you just revert this extra space?

Reply:
Resolved in 844fa1c. 😄